process/sync/shardblock_test.go

```diff
@@ -4165,11 +3903,119 @@ func TestBootstrap_SyncFromStorerShouldErrWhenLoadNotarizedBlocksFails(t *testin
 	assert.Equal(t, process.ErrNotEnoughValidBlocksInStorage, err)
 }
 
-func TestBootstrap_SyncFromStorerShouldWork(t *testing.T) {
+//func TestBootstrap_SyncFromStorerShouldWork(t *testing.T) {
```

Comment:
uncomment or remove this

Reply:
Fixed these tests, now they are uncommented